### ISSUE TITLE
"payload length" is not working

### ISSUE THREAD

**Issue by firatalkis** [ConSol-Monitoring/snclient#21]
Hi,

I have added tag of "payload length" under the " [/setting/NRPE/server] but ı got the following error.

"[2023-07-05 17:04:11.923][Error][pid:6248][listen_nrpe:80] nrpe protocol error: nrpe: response checksum failed: 3529001219 != 122204885"

![image](https://github.com/ConSol-Monitoring/snclient/assets/42132281/82e81c84-f30e-484e-9963-0c123e0016c6)

![image](https://github.com/ConSol-Monitoring/snclient/assets/42132281/a9025f5f-b536-40f6-b931-4c8853b38a13)

**Comment by firatalkis** [comment]
yes thats works thanks but if you have output size greater than 1024 byte I think the agent cutting the last character. do you have a comment about it?

**Comment by sni** [comment]
thanks, good catch, should be fine now.

**Comment by firatalkis** [comment]
> thanks, good catch, should be fine now.

Thanks, I will try it at next week.

**Comment by firatalkis** [comment]
> thanks, good catch, should be fine now.

do you have a chance tried this solution? because I tested version of 0.04 but problem is not fixed.

Still cutting last character when output size greather than 1024 byte.

![image](https://github.com/ConSol-Monitoring/snclient/assets/42132281/7175dfbc-71b3-42a5-a9e7-e8e92411a21a)

should be output:

![image](https://github.com/ConSol-Monitoring/snclient/assets/42132281/2f5823b9-204d-46ed-919a-fb0e9ec94a5a)

**Comment by sni** [comment]
you need to use a nightly build, this fix has not made it into a release yet.

**Comment by firatalkis** [comment]
oh, I am sorry :) thanks that worked
